### PR TITLE
Port problem.ctt

### PR DESCRIPTION
**Pull request by ecavallo**
From https://github.com/mortberg/cubicaltt/blob/pi4s3/examples/problem.ctt.

redtt gets stuck on `innerpath`. @mortberg

### CODE REVIEW

File: example/pointed.red [hunk 1]
```diff
@@ -46,4 +46,22 @@ let pf (pA : ptype) : pequiv (p→ pbool pA) pA =
         , λ j → bwdfwd/pt i j
         )
   in
-  (fwd, Iso/Equiv _ _ (fwd.0, bwd, λ a i → a, bwdfwd) .1)
+  (fwd, (Iso/Equiv _ _ (fwd.0, bwd, λ a i → a, bwdfwd).1))
```
Comment (favonia) [inline example/pointed.red]: There are some extra parentheses that I do not understand.

File: example/problem.red [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  )
+
+let ps2 : ptype = (s2, base)
+let ps3 : ptype = (s3, base)
+let pjoin : ptype = (join, inl base)
+
+let test0-2 : pΩ³ ps3 .0 =
+  λ i j k → cube i j k
+
+let f3 : (pΩ³ ps3 .0) → (pΩ³ pjoin .0) =
```
Comment (favonia) [inline example/problem.red]: You should not need these parentheses around the types unless you really want to. (I do not object to adding them, but want to know whether there are still bugs in the parser.)

File: example/pointed.red (hunk 1 above)
Comment (jonsterling) [reply example/pointed.red]: and let's put the space back before `.1`

**Comment by ecavallo** [comment]
OK, I guess we should merge this for now.

**Pull request merged**